pages/api/v1/utils/changeProfileScoreForAddress.ts

```
@@ -0,0 +1,52 @@
+		{
+			profile_score: firebaseAdmin.firestore.FieldValue.increment(change_score_by)
+		},
+		{
+			merge: true
+		}
+	);
+
+	// find user id for the address
+	const user_id = (await addressRef.get()).data()?.user_id;
+
+	if (isNaN(user_id)) return;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)